client/control/tests/components/RecipeForm.js

```diff
@@ -28,4 +28,22 @@ describe('<RecipeForm>', () => {
       <RecipeForm params={{}} location={{ query: '' }} />
     </Provider>);
   });
+
+  describe('changeAction', () => {
+    const store = controlStore();
+    const recipeFormWrapper = mount(<Provider store={store}>
+      <RecipeForm params={{}} recipe={{ action: 'show-heartbeat' }} location={{ query: '' }} />
+    </Provider>);
+
+    const disconnectedFormWrapper = recipeFormWrapper.find('DisconnectedRecipeForm').get(0);
+    disconnectedFormWrapper.changeAction({ target: { value: 'console-log' } });
+
+    it('should update the selectedAction in state', () => {
+      expect(disconnectedFormWrapper.state.selectedAction).toEqual({ name: 'console-log' });
```

Comment:
I kinda expected both of these assertions should be part of a single test, rather than split between two tests with a common `beforeEach` block.